### ISSUE TITLE
move tpe name up if it does not have "l" or "t" in it?

### ISSUE THREAD

**Issue by stefan11** [langsci/259#97]
![grafik](https://user-images.githubusercontent.com/4655859/109334176-a677e180-7860-11eb-9df3-632ca680be8d.png)

```latex
\avm{
[\type*{sign}
phonology & list!(phon\_string)! \\
synsem &	[local &	[category & category \\
						content & content \\
						qstore & set!(quantifier)! \\
						pool & set!(quantifier)! ] ] \\
retrieved & list!(quantifier)! ]
}
```
Would it be possible to have types like `sign` a bit higher up? Not for types including letters like `t` or `l`.

**Comment by kopeckyf** [comment]
I don't think this is a good idea. For one, consider what would happen if two types are placed in one line (e.g. two AVMs beside each other) and then the baselines of these types would not be identical. This would be terrible.

**Comment by stefan11** [comment]
@kopeckyf Could you please not close my bugs? Thanks.

Maybe both cases could be moved up a bit. I have to check.

**Comment by kopeckyf** [comment]
Maybe what you really want isn't so much messing around with individual letters, but rather setting tighter options for the enclosing bracket delimiter, e.g.: `\avm[delimfactor=950, delimfall=5pt] {}`.
